### ISSUE TITLE
Support VsCode Extensions

### ISSUE THREAD

**Issue by yekanchi** [madskristensen/VsixGallery#15]
I don't know how different are vscode `VSIX` Packages But I think it would be great if you add vsCode Extensions Support.

**Comment by reduckted** [comment]
I'm not sure exactly how much work it would be to support VS Code extensions, but that's only half the problem. The other half is that VS Code doesn't have native support for private extension galleries. 😢 

Check out this issue in the VS Code repo: https://github.com/Microsoft/vscode/issues/21839

Towards the bottom of that issue (from about [here onwards](https://github.com/Microsoft/vscode/issues/21839#issuecomment-609843029)) there are some solutions that people have come up with that you may find useful.

**Comment by yekanchi** [comment]
> I'm not sure exactly how much work it would be to support VS Code extensions, but that's only half the problem. The other half is that VS Code doesn't have native support for private extension galleries. 😢
> 
> Check out this issue in the VS Code repo: [microsoft/vscode#21839](https://github.com/microsoft/vscode/issues/21839)
> 
> Towards the bottom of that issue (from about [here onwards](https://github.com/Microsoft/vscode/issues/21839#issuecomment-609843029)) there are some solutions that people have come up with that you may find useful.

yes I'm aware of those but `https://open-vsx.org/` is written in java. i prefer C# solutions.

**Comment by madskristensen** [comment]
This request is out of scope for this project, I'm afraid. Closing
